### PR TITLE
Remove redis

### PR DESCRIPTION
**Pull request by apoorv1316**

### CODE REVIEW

File: scripts/benchmark.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Benchmark.ips do |x|
+  x.report("BulkInvoiceDownloadJob with Redis") do
+    BulkInvoiceDownloadJob.perform_now(
+      invoice_ids,
+      company_logo,
+      download_id,
+      root_url,
+      current_url_options
+    )
```
Comment (akhilgkrishnan) [inline scripts/benchmark.rb:20]: Instead of calling the Job, can you copy all the contents from the job and do a benchmarking.

Comment (apoorv1316) [reply scripts/benchmark.rb:20]: I did that and the results are same. Very minor difference. Should I go ahead and delete this file and uninstall redis?

**Comment by vipulnsward** [comment]
Instead of this, could we exploring removing BulkInvoiceDownloadChannel instead if possible?

We could remove ACa completely. 

If not, then we should explore anycable or using hotwire if possible, or just simple polling.

**Comment by vipulnsward** [comment]
@apoorv1316 what's the status on this? Can we wrap this up before picking up other issues?

**Comment by apoorv1316** [comment]
Closing this in favor of #1892

**Pull request closed without merging**